_posts/2019-03-06-cockpit-189.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Cockpit 189
+author: Gundersanne
+date: 2019-3-06
+tags: cockpit linux
+slug: cockpit-189
+category: release
+summary: Cockpit with Machines and Services improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 189.
+
+### Machines: Import existing image when creating VM
+
+Choose an existing disk image when create a new VM.
+
+![Create VM with existing image](/images/machines-existing-disk-image.png)
+
+### Machines: Introduce virtual networks
+
+Introduce read-only statistics for virtual networks.
```

Comment:
It's not really statistics, more like "Show the details and properties of virtual networks".